test/BuildingRegistry.Tests/BackOffice/Validators/PlanBuildingRequestValidatorTests.cs

```diff
@@ -23,8 +24,8 @@ public void GivenInvalidGeometry_ThenReturnsExpectedFailure()
             });
 
             result.ShouldHaveValidationErrorFor(nameof(PlanBuildingRequest.GeometriePolygoon))
-                .WithErrorCode(ValidationErrorCodes.Building.InvalidPolygonGeometry)
-                .WithErrorMessage(ValidationErrorMessages.Building.InvalidPolygonGeometry);
+                .WithErrorCode(ValidationErrors.Common.InvalidBuildingPolygonGeometry.Code)
```

Comment:
I don't like the tests using the codes instead of the actual error codes/messages.

Reply:
Agreed, refactor mistakes will be caught this way, and yes sometimes it's double the work if they need to change.

Reply:
Indeed, didn't notice this